app/questionnaire/placeholder_transforms.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from babel.numbers import format_currency
+from babel.dates import format_datetime
+from app.settings import DEFAULT_LOCALE
+
+
+class PlaceholderTransforms:
+    """
+    A class to group the transforms that can be used within placeholders
+    """
+    locale = DEFAULT_LOCALE
+    input_date_format = '%Y-%m-%d'
+
+    def format_currency(self, number=None):
+        return format_currency(number, 'GBP', locale=self.locale)
+
+    def format_date(self, date_to_format, date_format):
+        date_to_format = datetime.strptime(date_to_format, self.input_date_format)
+        return format_datetime(date_to_format, date_format, locale=self.locale)
+
+    @staticmethod
+    def filter_list(source_list, answer_ids, concatenate_with=None):
+        answer_list = []
+        source_list_by_group = sorted(source_list, key=lambda k: k['group_instance'])
+
+        for index in range(source_list_by_group[-1]['group_instance'] + 1):
+            answer_item = []
+            for answer in source_list_by_group:
+                if answer['group_instance'] == index and answer['answer_id'] in answer_ids:
+                    answer_item.append(answer['value'])
+
+            if concatenate_with:
+                answer_string = PlaceholderTransforms.concatenate_list(answer_item, concatenate_with)
+                answer_list.append(answer_string)
+            else:
+                answer_list.append(answer_item)
+
+        return answer_list
+
+    @staticmethod
+    def format_list(list_to_format):
+        formatted_list = '<ul>'
+        for item in list_to_format:
+            formatted_list += '<li>{}</li>'.format(item)
+        formatted_list += '</ul>'
+
+        return formatted_list
+
+    @staticmethod
+    def concatenate_list(list_to_concatenate, delimiter):
+        return delimiter.join(list_to_concatenate)
+
+    @staticmethod
+    def format_possessive(string_to_format):
```

Comment:
This should handle `'s` too shouldn't it?

Reply:
Yeah, seem to be missing tests for the transforms class too.